src/mds/Server.cc

```diff
@@ -5460,16 +5479,10 @@ int Server::parse_layout_vxattr(string name, string value, const OSDMap& osdmap,
     } else if (name == "layout.stripe_count") {
       layout->stripe_count = boost::lexical_cast<unsigned>(value);
     } else if (name == "layout.pool") {
-      try {
-	layout->pool_id = boost::lexical_cast<unsigned>(value);
-      } catch (boost::bad_lexical_cast const&) {
-	int64_t pool = osdmap.lookup_pg_pool_name(value);
-	if (pool < 0) {
-	  dout(10) << " unknown pool " << value << dendl;
-	  return -CEPHFS_ENOENT;
-	}
-	layout->pool_id = pool;
-      }
+      int64_t pool_id = parse_pool(value);
+      if (pool_id < 0)
+	return (int)pool_id;
```

Comment:
nit: indentation

Reply:
@vshankar The mis-indentation is most probably a visual issue. My Vim editor replaces 8 consecutive spaces with a `TAB` char and github probably doesn't care about that.